Sources/NIOHTTP2/HTTP2CommonInboundStreamMultiplexer.swift

```diff
@@ -93,7 +102,14 @@ extension HTTP2CommonInboundStreamMultiplexer {
             )
 
             self.streams[streamID] = channel
+
+            // If we have an async sequence of inbound stream channels yield the channel to it
+            // This also implicitly performs the stream initialization step
+            self.streamChannels?.yield(channel: channel.baseChannel)
```

Comment:
What does `channel.configureInboundStream` do  below? (Just checking we don't init the channel twice...)

Reply:
`channel.configureInboundStream` uses the `inboundStreamStateInitializer` `from the HTTP2Handler` which simply returns a void successful future.